setup.py

```diff
@@ -11,18 +11,11 @@
     install_requires=[
```

Comment:
Need to add `psutil`. Unless we remove `RequestsStreamFile` (which we can now for most platforms), we may also need to add `requests`.